api/swagger-spec/api.json

```diff
@@ -0,0 +1,27 @@
+{
+  "swaggerVersion": "1.2",
+  "apiVersion": "",
+  "basePath": "https://10.192.210.63:8443",
```

Comment:
> Unfortunately, it is.

If this is generated, you can likely get it to generate correctly by forcing `--listen` to "127.0.0.1".